backend/src/updatedIds.ts

```
@@ -37,3 +41,51 @@ try {
+    const requestInput = new RequestInput({id});
+    return [{index: "deces"}, buildRequest(requestInput)];
+  }).flat()}
+  const result =  await runBulkRequest(bulkRequest);
+  return result.responses
+}
+
+export const cleanRawUpdates = (rawUpdates: any, updates: any): Promise<Result> => {
+  return rawUpdates.map((r:any) => buildResultSingle(r.hits.hits[0]))
+    .filter((r:any) => Object.keys(r).length > 0)
+    .map((r:any) => {
+      delete r.score;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)